lib/hammer_cli/output/adapter/csv.rb

```diff
@@ -157,6 +157,7 @@ def print_record(fields, record)
     end
 
     def print_collection(fields, collection)
+      fields = displayable_fields(fields, collection.first)
```

Comment:
Similarly to my note in the base adapter. We shouldn't filter fields in printing collections in csv.